### DISCUSSION TITLE
AllTalk Version 2 - Screenshots

### DISCUSSION THREAD

**Discussion by erew123** [erew123/alltalk_tts#237] [General]
These are screenshots taken as I prepare to release the BETA of v2 of AllTalk within the next few days.

## Main Interface

![image](https://github.com/user-attachments/assets/59c08bb5-9873-4d96-aca8-f4b88aadfef8)

## Voice2RVC
![image](https://github.com/user-attachments/assets/2b38c4c2-e20e-4b50-a696-3968914ba0f3)

## Transcribe
![image](https://github.com/user-attachments/assets/89bb7305-f9ed-4806-9570-72fadf54c628)

## Dictate
![image](https://github.com/user-attachments/assets/80c317fc-9df5-4757-ad9b-5cd3db52385f)

## Global Settings

![image](https://github.com/user-attachments/assets/b0be44a9-6538-4c21-ac3d-cac428caa44a)

![image](https://github.com/user-attachments/assets/6afb9e2d-ab1f-427a-9331-40f603f5e5e7)

## RVC Settings
![image](https://github.com/user-attachments/assets/45a97982-4ca4-4654-a51f-788e4fb1bb67)

## TTS Engines management (1x example)
![image](https://github.com/user-attachments/assets/438219ed-7155-4976-832b-71eea2e1f226)

![image](https://github.com/user-attachments/assets/4b2dd912-2696-4990-950c-79d326d5eb04)

![image](https://github.com/user-attachments/assets/7ffc883d-4fa6-452b-946d-2fa7f7f8af8c)

![image](https://github.com/user-attachments/assets/55284a1a-772d-495d-8a10-c4a4e7bde83b)

## Documented throughout & on the GitHub Wiki

![image](https://github.com/user-attachments/assets/8dbbd436-3257-41ca-9f7c-117549db0f60)

## Terminal/Console output example

![image](https://github.com/erew123/alltalk_tts/assets/35898566/d4ff9068-190f-4ab5-889b-a0556893d963)

## XTTS Finetuning

![image](https://github.com/user-attachments/assets/b129924d-6e3c-4726-b34d-dc72f0d9e027)
![image](https://github.com/user-attachments/assets/297d1d79-97ea-4770-8c48-b5702699cc71)
![image](https://github.com/user-attachments/assets/95cd539a-9c1d-4a7a-b345-57b27128daf1)
![image](https://github.com/user-attachments/assets/8a7172e2-afb1-4098-82c1-5ee9c5e38e6c)
![image](https://github.com/user-attachments/assets/d653c586-562c-48b5-ba34-90e7ad96a28f)
![image](https://github.com/user-attachments/assets/650a77e7-7ab9-46c1-85f8-e79a68c2bb72)
![image](https://github.com/user-attachments/assets/44592699-8558-4066-b967-92058e6c22d7)

## Basic web interface

![image](https://github.com/erew123/alltalk_tts/assets/35898566/4390b52b-58de-4bc4-b379-0070868d15c3)

## TTS Generator

![image](https://github.com/user-attachments/assets/6cd013d3-1b8b-4f26-bb9b-a3c4f6d555fe)

## About

![image](https://github.com/user-attachments/assets/380e7360-621d-4300-86ee-1061e9a7b5bc)

## Text-gen-webui

![image](https://github.com/erew123/alltalk_tts/assets/35898566/5c64faf1-7f9c-4d7e-8ba6-df9e5d29a7b8)

## SillyTavern (RVC voice options & Silent Narrator options)

![image](https://github.com/erew123/alltalk_tts/assets/35898566/0f29659b-3868-4943-ad8d-9754d964c7eb)

**Comment by ghost** [comment]
The prompts are ignored when using Parler tts,is ther a fix for that?

**Reply by ghost** [reply]
for exmpale creating a a voice  in voce editor like this:

Talia speaks at an average pace with an animated delivery in a very confined sounding environment with clear audio quality.

shold use the same speaker every time but that prompt semms to be ignored

**Reply by erew123** [reply]
Hi @Larrycmd This is (to my knowledge) how Parler works. It literally generates a "new" voice each time. There is little to no-consistency in the voices. Other people comment on such an issue e.g. https://github.com/huggingface/parler-tts/issues/112

I have even asked the question about consistency of the supposed "built in voices" https://github.com/huggingface/parler-tts/issues/96

I also note this in the Engine information:

![image](https://github.com/user-attachments/assets/99c51430-7e02-4558-9fdd-8d207be18538)

To be clear, AllTalk is handing over the text you input to Parler's own scripts and its their own code that is generating the TTS. AllTalk is not manipulating/changing or altering anything and it is using Parler's suggested code implementation. I would suggest you add any comments/suggestions to the people at Parler https://github.com/huggingface/parler-tts and they can look to implement changes or better explain how the engine works (or doesnt work).

**Comment by RandomInternetPreson** [comment]
Holy guacamole! This looks cool!

**Comment by kiattisaknonsiri** [comment]
your code can do like rask.ai ?

**Reply by kiattisaknonsiri** [reply]
If so, How to do?

**Comment by KishoOoOoOo** [comment]
where is the XTTS Finetuning page 
i don't found it 
![386859416-b129924d-6e3c-4726-b34d-dc72f0d9e027](https://github.com/user-attachments/assets/8b10cc2d-ca70-4fb8-a109-e00278889873)

**Reply by RenNagasaki** [reply]
![grafik](https://github.com/user-attachments/assets/bd3c2eff-94e1-4fb6-80ce-148afac6883d)
The whole Process is XTTS. Just all the steps needed.

**Reply by KishoOoOoOo** [reply]
i dont have it 
![image](https://github.com/user-attachments/assets/99411a78-d1a0-4e21-9a54-32ae2e3f6b81)
 
 what should i do

**Reply by RenNagasaki** [reply]
Use the start_finetune script to start it. Not start_alltalk

**Reply by KishoOoOoOo** [reply]
I want finetune vits model, 😓

**Reply by KishoOoOoOo** [reply]
run with many error 
![image](https://github.com/user-attachments/assets/f204e962-370e-463d-a55e-1a0d2fcabb1c)
